pallets/identity/src/lib.rs

```diff
@@ -703,7 +703,7 @@ decl_module! {
         ) -> DispatchResult {
             let sender_key = AccountKey::try_from(ensure_signed(origin)?.encode())?;
             let from_did = Context::current_identity_or::<Self>(&sender_key)?;
-
+            ensure!(Self::is_auth_data_valid(&target, &authorization_data), Error::<T>::SignerNotMatchWithAuthData);
```

Comment:
I don't think we should be adding this. The way auths are structured is that anyone can authorize anyone for anything but the validity of the authorization is verified at the time acceptance of auth.